### DISCUSSION TITLE
I've adapted the profile loader for Linux

### DISCUSSION THREAD

**Discussion by jjramsey** [phnxdrgn536/FLSun-v400#26] [Ideas]
I modified the source of CuraProfileLoader so that it can be used on more traditional Unix-like systems. Currently, just Linux and FreeBSD are supported because that's what .NET supports. The files I changed were CopyService.cs and Directories.cs.

One change I made may positively affect Mac users as well, since there was some code that assumed that the backslash was the path separator, which isn't true on either MacOS or typical Unices. That's what the changes in Directories.cs were for.

If you like, I could send you a patch or the changed files, or I could create a pull request. Let me know what you prefer.

**Comment by phnxdrgn536** [comment]
if you want to go ahead and create a pull request i can give the files a look and if its merged it will put your name on the github as a contributor

**Reply by jjramsey** [reply]
I just made the pull request.
